### DISCUSSION TITLE
Welcome to OpenMultiBoxing Discussions!

### DISCUSSION THREAD

**Discussion by mooreatv** [OpenMultiBoxing/OpenMultiBoxing#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please note that for issues you can for now file them in the original project:

https://github.com/WowOpenBox/WowOpenBox/issues

**Comment by breezett93** [comment]
Does this work with other games besides WoW?

**Reply by mooreatv** [reply]
yes that is the whole point, it works with anything that you can open more than 1 copy/window of
